packages/plugin-tools/src/editors/ResourcesPicker.tsx

```diff
@@ -189,6 +189,13 @@ export const ResourcesPickerButton: React.FC<ResourcesPickerButtonProps> = props
       : typeof props.value === 'string'
       ? props.value
       : props.value?.options?.get(props.resourceName),
+    updateResourceId(newProps: ResourcesPickerButtonProps) {
+      this.resourceId = newProps.handleOnly
+      ? undefined
+      : typeof newProps.value === 'string'
```

Comment:
should it be `this.resourceId === newProps.handleOnly` ?

Reply:
No, we are updating the `this.resourceId`, given here : https://github.com/BuilderIO/builder/pull/3656/files#diff-34006edc80e20c930eeb7596aa28b3830fc551637be592c16edb6a7e76dab2e7L187